climate-data/climate-data/bin/slice.R

```diff
@@ -0,0 +1,17 @@
+library(tidyverse)
+
+args <- commandArgs(trailingOnly=TRUE)
+firstYear <- as.numeric(args[1])
+lastYear <- as.numeric(args[2])
+filenames <- args[c(-1, -2)]
+
+slice <- function(name, from, to) {
```

Comment:
There's already a function called slice in dplyr. Does that not do what you want?

Reply:
Not that this part is relevant to change, since I assume this is for getting data ready for the lesson.

Reply:
Yes, but I fixed it anyway - good catch.